### DISCUSSION TITLE
Ignore cache for specific packages

### DISCUSSION THREAD

**Discussion by OceanManOne** [python-poetry/poetry#7105] [Q&A]
Hi

I'm trying to achieve the following flow:
Installing pypi packages and installing local packages (using path = '../../lib1')

I'm not updating the versions of my local packages and so when I use poetry install (output everything to .venv directory to use in production) I want to always re-install my local packages as their code might changed, but not re-install pypi packages which shouldn't change.

Currently when I run poetry install 2nd time it says there is no dependencies to install or update even though I changed my local package (but didn't updated it's version).

Is there any way to force poetry to re-install certain package without removing the entire directory?
Also, I can't use develop = true because I'm outputting the poetry install output to a venv folder to be used in production.

**Comment by gudjonragnar** [comment]
Did you find any way to do this yet?
